.pre-commit-config.yaml

```diff
@@ -1,12 +1,7 @@
 files: 'pyogrio\/'
 repos:
-  - repo: https://github.com/psf/black
-    rev: 22.3.0
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: "v0.1.13"
     hooks:
-      - id: black
-        language_version: python3
-  - repo: https://github.com/pycqa/flake8
-    rev: 5.0.4
-    hooks:
-      - id: flake8
-        language: python_venv
+      - id: ruff
+      - id: ruff-format
```

Comment:
Should we put the formatter first? (eg I can imagine that the linter complains about a too long line, which would already be solved by the formatter afterwards)

Reply:
Sure; was just following the [example](https://github.com/astral-sh/ruff-pre-commit).

I'm assuming we don't want to auto-fix lint errors, which is where putting the linter first is recommended.